gatsby-config.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/camelcase */
+
+module.exports = {
+  plugins: [
+    "gatsby-plugin-postcss",
+    "gatsby-plugin-react-helmet",
+    "gatsby-plugin-styled-components",
+    "gatsby-plugin-typescript",
+
+    {
+      resolve: "gatsby-plugin-google-analytics",
+      options: {
+        trackingId: "UA-117977508-2",
```

Comment:
Note to self: to test this in staging, I'll need to temporarily disable the Google Analytics filter I set up which "blocks" localhost and now.sh sub-prod traffic.